aimmo-game-creator/service.py

```diff
@@ -8,9 +8,11 @@
 
 LOGGER = logging.getLogger(__name__)
 
+
 def create_game_rc(api, name, environment_variables):
     environment_variables['SOCKETIO_RESOURCE'] = "game/%s/socket.io" % name
-    environment_variables['GAME_API_URL'] = 'https://staging-dot-decent-digit-629.appspot.com/aimmo/api/games/'
+    environment_variables['GAME_API_URL'] =\
+        'https://staging-dot-decent-digit-629.appspot.com/aimmo/api/games/'
```

Comment:
I'm really not a fan of this syntax. Also, we don't care about the 80 character line limits in pep8